### PR TITLE
api-docs.yaml: adding number formats; making number integer where thi…

### PR DESCRIPTION
**Pull request by curoli**
api-docs.yaml: adding number formats; making number integer where this is clearly correct.

\<your comments for this PR go here\>

Have you read [CONTRIBUTING.md](../CONTRIBUTING.md) lately? If not, do that first.

I, the developer opening this PR, do solemnly pinky swear that:

- [ ] I've followed [the instructions](https://github.com/broadinstitute/firecloud-orchestration/blob/develop/CONTRIBUTING.md#api-changes) if I've made any changes to the API, _especially_ if they're breaking changes
- [ ] I've updated the RC_XXX release ticket with any manual steps required to release this change
- [ ] I've updated the [FISMA documentation](https://github.com/broadinstitute/firecloud-orchestration/blob/develop/CONTRIBUTING.md#fisma-documentation-changes) if I've made any security-related changes, including auth, encryption, or auditing

In all cases:

- [ ] Get two thumbsworth of review and PO signoff if necessary
- [ ] Verify all tests go green
- [ ] Squash and merge; you can delete your branch after this **unless it's for a hotfix**. In that case, don't delete it!
- [ ] Test this change deployed correctly and works on dev environment after deployment

### CODE REVIEW

File: src/main/resources/swagger/api-docs.yaml [hunk 1]
```diff
@@ -7457,6 +7503,7 @@ definitions:
             default: broad-dsde-dev,
           methodVersion:
             type: integer
+            format: int64
```
Comment (davidangb) [inline src/main/resources/swagger/api-docs.yaml]: snapshotIDs are elsewhere defined as `int32`s

**Review by davidangb — COMMENTED** [review]
one minor comment, otherwise looks good!

File: src/main/resources/swagger/api-docs.yaml (hunk 1 above)
Comment (curoli) [reply src/main/resources/swagger/api-docs.yaml]: Thanks, good catch! I committed a fix.

**Review by davidangb — APPROVED** [review]
:+1: thanks again! Merge at will.

**Comment by curoli** [comment]
I think I can't merge, because I don't have write access.

**Comment by davidangb** [comment]
ah, of course. I'll push it through then.

**Pull request merged**